packages/player/src/components/video.tsx

```diff
@@ -93,6 +93,9 @@ export const Video: React.FC<VideoProps> = ({
       onPlaying={() => {
         videoService.send('DONE_WAITING')
       }}
+      onEnded={() => {
```

Comment:
this is the browser api for an ended event. dunno how consistent it is.

https://github.com/joelhooks/cueplayer-react/blob/main/src/components/Video.js#L559

it's what the currently deployed one uses